tests/foreman/cli/test_docker.py

```diff
@@ -138,7 +138,7 @@ def test_create_one_docker_repo(self):
 
     @tier1
     @run_only_on('sat')
-    def test_create_multiple_docker_repo(self):
+    def test_positive_create_multiple_docker_repos_using_same_product(self):
```

Comment:
Use plural instead of word `multiple` (if we're not talking about multiple publishing/promoting, for example). And no need in word `docker` as it's `docker` module and `DockerRepositoryTestCase` class, so it should be just `test_positive_create_repos_using_same_product`